packages/beacon-node/src/network/networkConfig.ts

```diff
@@ -10,12 +10,12 @@ import {NodeId, computeNodeId} from "./subnets/interface.js";
 export class NetworkConfig {
   private readonly nodeId: NodeId;
   private readonly config: BeaconConfig;
-  private custodyConfig: CustodyConfig;
+  readonly custodyConfig: CustodyConfig;
```

Comment:
ahhh... was reviewing the files in order and see that you changed this.  I think we should use the getter pattern and leave this `private readonly`
```suggestion
  private readonly custodyConfig: CustodyConfig;
```

Reply:
sounds good! pushed an update to use the getter pattern